server/controllers/manage/bookingsController.ts

```diff
@@ -19,16 +20,37 @@ export default class BookingsController {
     }
   }
 
-  new(): RequestHandler {
-    return (req: Request, res: Response) => {
+  find(): RequestHandler {
```

Comment:
Sorrry to be annoying, but I think this needs to be in `new`. We can then define the template based on what is in the flash.